src/main/resources/com/ncc/aif/ontologies/README.md

```diff
@@ -0,0 +1,64 @@
+# AIDA Ontologies
+
+The following is a listing of the ontologies found at AIDA-Interchange-Format/src/main/resources/com/ncc/aif/ontologies.
+Each section in this README contains a short description of what they are built from, what they are used for,
+and any other points of relevant information that might be of some use to individuals using these files.
+
+## Differences Between *Ontology and *OwlOntology Files
+
+Some ontologies have a corresponding file with a similar name such as SeedlingOntology and SeedlingOwlOntology. In these
+cases, the *OwlOntology file uses strict OWL and won't include third-party properties/classes
+like schema:rangeIncludes. The *Ontology files are not restricted and make use of third-party properties/classes to simplify definitions.
+Both the *OwlOntology and *Ontology files define the same vocabulary.
+
+# AIF Ontology Files
+
+### AidaDomainOntologiesCommon
+
+A collection of top level classes all Domain Ontologies reference.  Additionally, there are
+also a few universal classes that include important properties such as the CanHaveNumericValue class.  This ontology is
+a hand-written source file and is not generated.
+
+### InterchangeOntology
+
+This ontology contains the Interchange vocabulary and is written in strict OWL. In particular, it contains
```

Comment:
After discussion with @jakevb, I think I understand where I was confused.  I recommend changing the entire sentence starting from "In particular..." to the following.

It defines the structure of the vocabulary, including the hierarchy of terms (via the "domain" schema definition) and the possible datatypes of terms (via the "range" schema definition).

@ecurley-nextcentury - do you agree with the rewording of this?

Reply:
All RDF is represented as triples (**subject**, **predicate**, **object**). "Domain" refers to what types a particular **predicate** can have as **subject** and "range" refers to what types that **predicate** can have as **object**. If these terms are confusing things, we should just remove them. We can probably change the description to something like, "This ontology defines the Interchange vocabulary and how the defined classes and properties may interact". I'll leave it to you and Jake to come up with something that works. Let me know if you have any questions.

Reply:
Your recommendation is good with me.

Reply:
Sounds good.  I incorporated Eddie's suggested change.